app/lib/common/utils/utils.dart

```diff
@@ -290,3 +291,8 @@ const List<(RSVP, String)> rsvpOptions = <(RSVP, String)>[
   (RSVP.Maybe, 'Maybe'),
   (RSVP.Yes, 'Yes'),
 ];
+
```

Comment:
uhh... I like that. Very nice!

Reply:
could place them in providers folder but looks cleaner here. Should move it?